src/_repobee/config.py

```diff
@@ -126,8 +124,8 @@ def _read_config(config_file: pathlib.Path) -> configparser.ConfigParser:
 
     if constants.CORE_SECTION_HDR not in config_parser:
         raise exception.FileError(
-            "config file at '{!s}' does not contain the required "
-            "[repobee] header".format(config_file)
+            f"config file at '{str(config_file)}' does not contain the "
```

Comment:
Just as a future note, there's never a need to call `str` on an argument in an f-string, that's implicit. You probably did it here because there (for some unexplained reason) is a `!s` format directive in some of the the placeholders of the format strings, and you have a keen eye for details. But that format directive was also redundant, not sure why it was there.

Reply:
Yeah, there was a `!s` within the format directive, and that within prior PRs to replace str.format with f-strings, the `str()` cast was kept in this same format, so I figured it was desired to keep.

Reply:
Yeah, that's totally understandable. It's been a bit inconsistent. I made a note in #460 that one does not need to put in the `str()` call into an f-string.